internal/core/graphics/image.rs

```diff
@@ -1324,6 +1324,51 @@ pub(crate) mod ffi {
     ) {
         image.set_nine_slice_edges(top, right, bottom, left);
     }
+
+    #[no_mangle]
+    pub unsafe extern "C" fn slint_image_to_rgb8(
```

Comment:
technically, these function don't need to be unsafe. They don't have invariant that is not expressed in the types.

Reply:
Ahh, copy & paste :). I'll remove it.